tests/31sync/06state.pl

```diff
@@ -542,6 +560,10 @@ sub wait_for_event_in_room {
             state_key => $user->user_id,
          );
       })->then( sub {
+         matrix_send_room_text_message_and_wait_for_sync( $user, $room_id,
```

Comment:
If tracking down this bug is going to hold up progress too long, could you at least put a `TODO(mjark)` marker in somewhere, so we can hunt that down later?